website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,129 @@
+See [Provisioner CRD](/docs/provisioner-crd/) for a description of settings.
+* **Well-known labels**: The CRD can use well-known Kubernetes labels to allow pods to request only certain instance types, architectures, operating systems, or other attributes when creating nodes.
+See [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) for details.
+Keep in mind that only a subset of these labels are supported in Karpenter, as described in [Provisioning](/docs/concepts/provisioner.md).
+* **Time to live**: A provisioner can also include time-to-live values to indicate when nodes should be decommissioned after a set amount of time from when it was created or after it becomes empty of deployed pods.
+* **Multiple provisioner CRDs**: An important feature of Karpenter is that it allows multiple provisioner CRDs on the same cluster.
+One possible result is that you could configure different teams on the same cluster to run on completely separate capacity.
+One team could run on GPU nodes while another uses storage nodes, with each having a different set of defaults.
+Although you can address the needs of multiple teams in one CRD in some ways, having separate ones lets you isolate nodes for billing, use different defaults (such as no GPUs for a team), or use different expiration and scale-down values.
+Having multiple CRDs could some day lead Karpenter to allow provisioners with different cloud providers on the same cluster or a control plane on one cloud and data plane on another.
+
+## Scheduling
+
+Besides creating nodes, Karpenter also schedules pods.
+To avoid conflicts, Karpenter only schedules pods that the Kubernetes scheduler has marked unschedulable.
+Karpenter optimistically creates the Node object and binds the pod.
+This stateless approach avoids race conditions and improves performance.
+If something is wrong with the launched node, Kubernetes will automatically migrate the pods to a new node.
+
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+This is useful if there is additional room in the node due to imperfect packing shape or as workloads finish over time.
+
+## Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to work other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set some values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+
+Choosing spot instances is one feature that is specific to AWS.
```

Comment:
This isn't actually accurate. I'd remove it. We're trying to generalize this concept, but just haven't done it yet.

Reply:
I'll not that it's AWS for now but could be generalized in the future.